cmd/gateway/manta/gateway-manta.go

```diff
@@ -0,0 +1,498 @@
+/*
+ * Minio Cloud Storage, (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
```

Comment:
Why is the package name `cmd` and also importing `cmd` as `minio`? Shouldn't it be `manta` or something? Azure looks to be using `azure` under their new modular gateway directory.

I only noticed this because it's confusing `godef`. 😉

Reply:
Yes it should fixed to manta.

Reply:
Second thought, I'm not sure why `godef` won't find symbols under that new import. Still curious about the naming convention though.

Reply:
@harshavardhana I can take care of that.